lambdas/revoke-vcs/src/main/java/uk/gov/di/ipv/core/revokevcs/RevokeVcsHandler.java

```diff
@@ -87,21 +90,25 @@ public void handleRequest(InputStream inputStream, OutputStream outputStream, Co
                         String.format("Revoking %s VCs.", userIdCriIdPairs.size())));
 
         try {
-            revoke(userIdCriIdPairs);
+            var results = revoke(userIdCriIdPairs);
             LOGGER.info(
                     LogHelper.buildLogMessage(
                             String.format(
                                     "Finished attempt to revoke %s VCs.",
                                     userIdCriIdPairs.size())));
+
+            objectMapper.enable(SerializationFeature.INDENT_OUTPUT);
+            outputStream.write(objectMapper.writeValueAsString(results).getBytes());
```

Comment:
I think you might be able to write directly to the output stream - try `objectMapper.writeValue(outputStream, results)`